openupgradelib/openupgrade.py

```diff
@@ -960,19 +962,34 @@ def reactivate_workflow_transitions(cr, transition_conditions):
             (condition, transition_id))
 
 
-def migrate(no_version=False):
+def migrate(no_version=False, v9env=False, context=None, *args, **kwargs):
```

Comment:
Good idea, thought about something even more versatile, but didn't come up with the idea to set it `None`.